src/main/resources/org/jfrog/hudson/pipeline/scripted/steps/ArtifactoryGradleBuild/help-switches.html

```diff
@@ -0,0 +1,3 @@
+<div>
+    Specify a list of Gradle switches to be invoked, or leave it empty.
```

Comment:
You can mention that this is "optional" in case someone thinks they have to include the switch information

Reply:
That's noted @Kwhetstone Thanks.